query/src/main/scala/filodb/query/exec/MetadataExecPlan.scala

```diff
@@ -474,13 +486,28 @@ final case object TsCardExec {
     prefix.mkString(PREFIX_DELIM).utf8
   }
 
-  case class CardCounts(active: Int, total: Int) {
+  /**
+   * @param prefix ShardKeyPrefix from the Cardinality Record
+   * @param datasetName FiloDB dataset name
+   * @return concatenation of ShardKeyPrefix and filodb dataset
+   */
+  def prefixToGroupWithDataset(prefix: Seq[String], datasetName: String):ZeroCopyUTF8String = {
+    s"${prefix.mkString(PREFIX_DELIM)}$PREFIX_DELIM$datasetName".utf8
+  }
+
+  /**
+   * @param active Actively (1 hourt) Ingesting Cardinality Count
+   * @param total aka shortTerm. This is the 7 day running Cardinality Count
```

Comment:
I might have already asked this: can we rename this now? We can leave "total" in e.g. the V1 JSON, but we might as well rename variables and V2 JSON keys (as long as that isn't a ton of work to pull off).

Reply:
Surely, I forgot about this. updated